### PR TITLE
Do compaction of JSON-LD documents

### PR DESCRIPTION
**Pull request by Tpt**

### CODE REVIEW

File: src/JsonLdResourceNode.php [hunk 1]
```diff
@@ -73,4 +72,22 @@ private function getUris() {
 
 		return $uris;
 	}
+
+	private function setJsonLdDocumentLoader() {
+		global $jsonld_default_load_document;
```
Comment (ProgVal) [inline src/JsonLdResourceNode.php]: Where is this variable used?

Comment (Tpt) [inline src/JsonLdResourceNode.php]: In the lib. Yes, it is ugly but it's the best lib I've found.

Comment (ProgVal) [inline src/JsonLdResourceNode.php]: It lacks of some factories.

Comment (Tpt) [inline src/JsonLdResourceNode.php]: Yeah

**Pull request merged**